src/components/Header.js

```diff
@@ -1,7 +1,22 @@
+import Link from 'next/link';
+import styled from 'styled-components';
+
+const Container = styled.div`
```

Comment:
You don't need an extra `div` here. You could style the `header` directly.

Reply:
fixed.